.github/workflows/reference_linter.yaml

```diff
@@ -3,14 +3,14 @@ on:
   push:
     paths:
       - 'locales/en/**.ftl'
-      - 'pendingTranslations.ftl'
+      - 'locales-pending/*.ftl'
     branches:
       - main
       - localization
```

Comment:
We should remove this, it's not used anymore

Reply:
```suggestion
```